src/core/Table/TableRowMemoized.tsx

```diff
@@ -58,8 +61,25 @@ const TableRow = <T extends Record<string, unknown>>(props: {
           );
         })}
       </div>
-      {row.isExpanded && subComponent && (
-        <div className='iui-row iui-expanded-content'>{subComponent(row)}</div>
+      {subComponent && (
+        <CSSTransition
+          in={row.isExpanded}
+          timeout={200}
+          unmountOnExit={true}
+          onEnter={(node) => {
+            if (expandedHeight === 0) {
+              setExpandedHeight(node.getBoundingClientRect().height);
```

Comment:
@bentleyvk Do you know a better way to "save" the original height before setting inline height to 0?

Reply:
Maybe from the ref?

Reply:
I see Material-UI is using wrapper ref for this:
https://github.com/mui-org/material-ui/blob/e724d98eba018e55e1a684236a2037e24bcf050c/packages/material-ui/src/Collapse/Collapse.js#L167

Reply:
They have like 5 different wrappers 😄 

But yeah, I'll try with ref